### PR TITLE
Release capa version 7.0.0

### LINKED ISSUE #1963: Fix linter script issues
see https://github.com/mandiant/capa-rules/actions/runs/7725366127/job/21059427544

```
ERROR:lint:failed to extract capabilities: create directory /home/runner/work/capa-rules/capa-rules/tests/data/dotnet/692f7fd6d198e804d6af98eb9e390d61.exe_ unexpected format: dotnet
Traceback (most recent call last):
  File "/home/runner/work/capa-rules/capa-rules/scripts/lint.py", line 419, in check_rule
    capabilities = get_sample_capabilities(ctx, path)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/capa-rules/capa-rules/scripts/lint.py", line 368, in get_sample_capabilities
    extractor = capa.loader.get_extractor(
                ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/capa-rules/capa-rules/capa/loader.py", line 255, in get_extractor
    vw = get_workspace(input_path, input_format, sigpaths)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/capa-rules/capa-rules/capa/loader.py", line 157, in get_workspace
    raise ValueError("unexpected format: " + input_format)
ValueError: unexpected format: dotnet
```

### PR DESCRIPTION
**Pull request by mr-tz**
# Release checklist

- [x] Ensure all [milestoned issues/PRs](https://github.com/mandiant/capa/milestones) are addressed, or reassign to a new milestone.
  - [x]  merge #1962 
  - [x] fix #1963
- [x] Add the `dont merge` label to all PRs that are close to be ready to merge (or merge them if they are ready) in [capa](https://github.com/mandiant/capa/pulls) and [capa-rules](https://github.com/mandiant/capa-rules/pulls).
- [x] Ensure the [CI workflow succeeds in master](https://github.com/mandiant/capa/actions/workflows/tests.yml?query=branch%3Amaster).
- [x] Ensure that `python scripts/lint.py rules/ --thorough` succeeds (only `missing examples` offenses are allowed in the nursery). You can [manually trigger a thorough lint](https://github.com/mandiant/capa-rules/actions/workflows/tests.yml) in CI via the "Run workflow" option. 
  - [x] https://github.com/mandiant/capa-rules/actions/runs/7739872595/
- [x] Review changes
  - capa https://github.com/mandiant/capa/compare/\<last-release\>...master
  - capa-rules https://github.com/mandiant/capa-rules/compare/\<last-release>\...master
- [x] Update [CHANGELOG.md](https://github.com/mandiant/capa/blob/master/CHANGELOG.md)
  - Do not forget to add a nice introduction thanking contributors
  - Remember that we need a major release if we introduce breaking changes
  - Sections: see template below
  - Update `Raw diffs` links
  - Create placeholder for `master (unreleased)` section
    ```
    ## master (unreleased)

    ### New Features

    ### Breaking Changes

    ### New Rules (0)

    -

    ### Bug Fixes

    ### capa explorer IDA Pro plugin

    ### Development

    ### Raw diffs
    - [capa <release>...master](https://github.com/mandiant/capa/compare/<release>...master)
    - [capa-rules <release>...master](https://github.com/mandiant/capa-rules/compare/<release>...master)
    ```
- [x] Update [capa/version.py](https://github.com/mandiant/capa/blob/master/capa/version.py)
- [x] Create a PR with the updated [CHANGELOG.md](https://github.com/mandiant/capa/blob/master/CHANGELOG.md) and [capa/version.py](https://github.com/mandiant/capa/blob/master/capa/version.py). Copy this checklist in the PR description.
- [ ] After PR review, merge the PR and [create the release in GH](https://github.com/mandiant/capa/releases/new) using text from the [CHANGELOG.md](https://github.com/mandiant/capa/blob/master/CHANGELOG.md).
- Verify GH actions
  - [ ] [upload artifacts](https://github.com/mandiant/capa/releases)
  - [ ] [publish to PyPI](https://pypi.org/project/flare-capa)
  - [ ] [create tag in capa rules](https://github.com/mandiant/capa-rules/tags)
  - [ ] [create release in capa rules](https://github.com/mandiant/capa-rules/releases)
- [ ] [Spread the word](https://twitter.com)
- [ ] Update internal service

### CODE REVIEW

**Comment by williballenthin** [comment]
shall we sync with master so we can pull in the latest Ghidra UI features?

**Comment by mr-tz** [comment]
Yes, I just created this branch for overall tracking. We should get all GSoC+ work and related ready items in v7!

**Comment by williballenthin** [comment]
please hold off merging/releasing pending some internal discussion about a further change to the proto format.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,13 +4,11 @@
...
 
-- add Ghidra UI integration #1734 @colton-gabertan @mike-hunhoff
+- 
 
 ### Breaking Changes
 
-- main: introduce wrapping routines within main for working with CLI args #1813 @williballenthin
-- move functions from `capa.main` to new `capa.loader` namespace #1821 @williballenthin
-- proto: add `package` declaration #1960 @larchchen
+-
```
Comment (mr-tz) [inline CHANGELOG.md:11]:
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,13 +4,10 @@
 
 ### New Features
 
-- add Ghidra UI integration #1734 @colton-gabertan @mike-hunhoff
+-
```
Comment (mr-tz) [inline CHANGELOG.md:7]:
```suggestion
```

**Comment by mr-tz** [comment]
this should be ready to rock and roll

File: CHANGELOG.md [hunk 3]
```diff
@@ -23,16 +19,19 @@
...
+- [capa-rules v7.0.0...master](https://github.com/mandiant/capa-rules/compare/v7.0.0...master)
+
+## v7.0.0
+This is the v7.0.0 release of capa which was mainly worked on during the Google Summer of Code (GSoC) 2023. A huge
+shoutout to our GSoC contributors @colton-gabertan and @yelhamer for their amazing work.
 
-## v7.0.0-beta
-This is the beta release of capa v7.0 which was mainly worked on during the Google Summer of Code (GSoC) 2023. A huge
-shoutout to @colton-gabertan and @yelhamer for their amazing work.
+Also, a big thanks to the other contributors: @aaronatp, @Aayush-Goel-04, @bkojusner, @doomedraven, @ruppde, and @xusheng6.
```
Comment (williballenthin) [inline CHANGELOG.md:29]:
```suggestion
Also, a big thanks to the other contributors: @aaronatp, @Aayush-Goel-04, @bkojusner, @doomedraven, @ruppde, @larchchen, @JCoonradt, and @xusheng6.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -23,16 +19,19 @@
...
 
 ### Raw diffs
-- [capa v7.0.0-beta...master](https://github.com/mandiant/capa/compare/v7.0.0-beta...master)
-- [capa-rules v7.0.0-beta...master](https://github.com/mandiant/capa-rules/compare/v7.0.0-beta...master)
+- [capa v7.0.0...master](https://github.com/mandiant/capa/compare/v7.0.0...master)
+- [capa-rules v7.0.0...master](https://github.com/mandiant/capa-rules/compare/v7.0.0...master)
+
+## v7.0.0
+This is the v7.0.0 release of capa which was mainly worked on during the Google Summer of Code (GSoC) 2023. A huge
+shoutout to our GSoC contributors @colton-gabertan and @yelhamer for their amazing work.
```
Comment (williballenthin) [inline CHANGELOG.md:27]: lets add links to blog posts when they go live.

ditto with the main readme.

**Review by williballenthin — APPROVED** [review]

File: CHANGELOG.md (hunk 4 above)
Comment (mr-tz) [reply CHANGELOG.md:27]: yes, I will update the Release page with the links

**Pull request merged**